public/js/leaderboard.js

```diff
@@ -2,18 +2,51 @@
 $('#nav-leaderboard').addClass('active');
 
 /* Get HTML for the complete leaderboard table from the server and display it. */
-var fetchLeaderboard = function() {
+var fetchLeaderboard = function(t) {
+    
     $.ajax({
         type: 'GET',
         url: '/leaderboard-table',
         data: {
             fullTable: true,
-            longTable: true
+            longTable: true,
+            type: t
         },
         success: function(data) {
             $('.leaderboard-content').html(data);
         }
     });
 }
 
-fetchLeaderboard();
+fetchLeaderboard('overall');
+
+$('#option-overall').click(function(evt) {
+    $('#points-board-card').addClass('hidden');
+    $('#accuracy-board-card').addClass('hidden');
+    $('#overall-board-card').removeClass('hidden');
+    fetchLeaderboard('overall');
+});
+
+$('#option-points').click(function(evt) {
+    $('#overall-board-card').addClass('hidden');
```

Comment:
this section of code seems to be repeated a lot, cant we group this in a function?

Reply:
Good point, I'll try to goup them.